### PR TITLE
Fix anchor and link for re-using SSH connections

### PR DESCRIPTION
**Pull request by jasoncpatton**

### CODE REVIEW

File: uw-research-computing/connecting.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ throughput or high performance compute systems. If you haven\'t, see our
 2.  [Logging In](#login)
     -   [On the command line (Mac/Linux)](#login-ssh)
     -   [Using an SSH program (Windows/Mac)](#login-putty)
-    -   [Logging in Automatically](#ssh-keys)
+    -   [Re-using SSH connections](#reuse-ssh)
```
Comment (CannonLock) [inline uw-research-computing/connecting.md:17]:
```suggestion
    -   [Re-using SSH connections](#c-transferring-multiple-files)
```

File: uw-research-computing/connecting.md [hunk 2]
```diff
@@ -125,15 +125,19 @@ this guide](http://www.howtogeek.com/66776/how-to-remotely-copy-files-over-ssh-w
 for instructions on how to do this, starting at the section titled
 **\"SSH and SCP Without Passwords\"**. -->
 
+<a name="reuse-ssh"></a>
```
Comment (CannonLock) [inline uw-research-computing/connecting.md:128]:
```suggestion
```

**Review by CannonLock — CHANGES_REQUESTED** [review]
Jekyll adds ids to header automatically. It is easier to use these to navigate.

File: uw-research-computing/connecting.md (hunk 1 above)
Comment (jasoncpatton) [reply uw-research-computing/connecting.md:17]: How are these generated? Should this actually be:

```suggestion
    -   [Re-using SSH connections](#c-re-using-ssh-connections)
```

Comment (ChristinaLK) [reply uw-research-computing/connecting.md:17]: @jasoncpatton I think you're right. everything is lowercase and all spaces/punctuation replaced with a dash.

Comment (jasoncpatton) [reply uw-research-computing/connecting.md]: Ok, I'll commit these and we'll fix later if they're wrong.

**Review by ChristinaLK — APPROVED** [review]

**Review by CannonLock — APPROVED** [review]
Tested locally and works as expected!

**Comment by jasoncpatton** [comment]
Thanks @ChristinaLK @CannonLock !!!

**Pull request merged**